ci/bootstrap-env/src/main.rs

```diff
@@ -59,7 +59,7 @@ fn main() -> Result<()> {
         automatically_add_cw20s: false,
         automatically_add_cw721s: false,
         voting_module_instantiate_info: ModuleInstantiateInfo {
-            code_id: orc.contract_map.code_id("cwd_voting_cw20_staked")?,
+            code_id: orc.contract_map.code_id("cw20_staked_balance_voting")?,
```

Comment:
think if you rebase on main the new name should work now

Reply:
bunch of these naming rebase hell changes in here, won't add comment for all of them :)

Reply:
uh oh